### PR TITLE
Fix minimum as zero case.

### LINKED ISSUE #5: Slider filter says "not enough data" when there is a 0 value
I have a collection of entities with a value that can have value 0, and would wish to filter them using this utility. However, when one has value 0, the slider disappears and says there's not enough data. In the example shown below, if I change the second entity's "Coste de ingeniería" value to any positive value, the filter works as intended.
Anotación 2019-08-20 113626

### PR DESCRIPTION
**Pull request by jacqueswww**
Fixes #5 

Basically 0 was evaluated as falsy when in this case zero is a valid value for minimum.

### CODE REVIEW

File: admin_numeric_filter/templates/admin/filter_numeric_slider.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
         <h3>{% blocktrans with filter_title=title %}By {{ filter_title }}{% endblocktrans %}</h3>
 
-        {% if choice.min and choice.max and choice.step %}
+        {% if choice.min or choice.min == 0 choice.min and choice.max and choice.step %}
```
Comment (tuffnatty) [inline admin_numeric_filter/templates/admin/filter_numeric_slider.html]: Hey, did you test that line? ;-) there is definitely too much choice.min

**Review by tuffnatty — CHANGES_REQUESTED** [review]

File: admin_numeric_filter/templates/admin/filter_numeric_slider.html (hunk 1 above)
Comment (lukasvinclav) [reply admin_numeric_filter/templates/admin/filter_numeric_slider.html]: Maybe we should do the same check with the max value as well. What do you think?

Comment (jacqueswww) [reply admin_numeric_filter/templates/admin/filter_numeric_slider.html]: :+1:  Please check now.

Comment (lukasvinclav) [reply admin_numeric_filter/templates/admin/filter_numeric_slider.html]: What about this?

`{% if choice.min is not None and choice.max is not None and choice.step %}`

**Comment by Qu4tro** [comment]
Is there anything remaining for this PR?

**Comment by tuffnatty** [comment]
> Is there anything remaining for this PR?

@lukasvinclav 's suggestion seems to be cleaner:
```if choice.min is not None and choice.max is not None and choice.step```
It avoids extra bool casting and ambiguous logical operator precedence.

**Comment by Qu4tro** [comment]
@jacqueswww would you mind if I pushed the alternative version as a PR?

**Comment by jacqueswww** [comment]
@Qu4tro sure; `if choice.min is not None and choice.max is not None and choice.step` does seem cleaner.

**Comment by nabinkhadka** [comment]
@jacqueswww Just installed this library, but I am still getting the same message "No enough data.". Is this just because there are value 0 (a lot of them in my case)?

**Pull request closed without merging**